app/src/main/res/drawable/layout_background_rounded.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<shape xmlns:android="http://schemas.android.com/apk/res/android">
+    <solid android:color="@color/white"/>
+    <stroke android:width="2dp" android:color="#B1BCBE" />
+    <corners android:radius="10dp"/>
+    <padding android:left="0dp" android:top="0dp" android:right="0dp" android:bottom="0dp" />
+</shape>
```

Comment:
@so7fie I honestly don't know because I don't know the exact definition of a theme. This is basically a custom background for each sale (the `CardView` I mean). Do you think it should be put somewhere else?

Reply:
I cannot tell you exactly how to create a theme, but they are present in xml files in the `res` package. It is not necessary to put this as a part of a theme, but it would be cleaner and more elegant. 

Maybe @Huntsekkern could answer it better than me, since he did the theme for the buttons.

Reply:
It is certainly not a theme as a theme defines a color scheme for the app.
I think @so7fie is mistaking a theme and a style.
A style is a set of attributes (colours, margins, size, etc) for a repeatable UI element such as a button.
I think a style must have a parent from the Palette though (such as a button, a radio box, a check box, a text view, ...). If you're creating a UI element from scratch, it seems pretty sensible to me to classify it in drawable.